frontend/src/features/properties/components/LastUpdatedBy.tsx

```diff
@@ -14,35 +14,43 @@ const UpdateText = styled.p`
   padding: 1rem;
   display: flex;
   justify-content: flex-end;
+  margin-bottom: 0px;
 `;
 
 const LinkButton = styled.span`
-  background: none;
-  border: none;
-  padding: 0;
-  color: #069;
+  color: #18548c;
   text-decoration: underline;
 `;
 
 /**
  * Display formatted last update user name, email, and date/time.
  */
-const LastUpdatedBy: React.FunctionComponent<ILastUpdatedByProps> = ({
+const LastUpdatedBy: React.FC<ILastUpdatedByProps> = ({
   createdOn,
   updatedOn,
   updatedByName,
   updatedByEmail,
 }) => {
-  return updatedOn || createdOn ? (
-    <UpdateText>
-      Last Updated By:&nbsp;
-      <TooltipWrapper toolTipId="lastUpdatedEmail" toolTip={updatedByEmail ?? 'unknown'}>
-        <LinkButton>{updatedByName ?? 'unknown'}</LinkButton>
-      </TooltipWrapper>
-      &nbsp;
-      {formatApiDateTime(updatedOn ?? createdOn)}
-    </UpdateText>
-  ) : null;
+  if (updatedOn) {
+    return (
+      <UpdateText>
+        Last Updated By:&nbsp;
+        <TooltipWrapper toolTipId="lastUpdatedEmail" toolTip={updatedByEmail}>
```

Comment:
If I remember correctly, part of the reason for the 'unknown' is that we don't have a firm requirement on users in the system having an email associated, or even a name. Those are both optional fields.

Reply:
Ah, I gotcha. I'll put that back